test/unit/sth_test.js

```diff
@@ -324,13 +324,45 @@ describe('sth tests', function() {
             })
         );
 
+        it(
```

Comment:
I'd suggest to add a couple of cases:

* should respond with 400 - Bad Request if aggMethod is not a valid one (e.g. aggMethod=foo)
* should respond with 400 - Bad Request if aggMethod mixes a valid one with a not valid one (eg. aggMethod=max,foo)

Btw, what happens if aggMethod mixes all with another method? Eg. aggMethod=all,min? Do we get and error or the "min" part is ignored and that's equivalent to aggMethod=all?

Reply:
Added test cases in [3b51b4c](https://github.com/telefonicaid/fiware-sth-comet/pull/605/commits/3b51b4c42e14b77f331525db1fcd0926a79afffd)

If aggMethod mixes `all` with another method e.g. `aggMethod=all,min`, then it is equivalent to `aggMethod=all` ignoring all other parameters.